[2025-02-10T21:26:05.226+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T21:26:05.393+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.check_file_in_directory scheduled__2025-02-10T21:25:00+00:00 [queued]>
[2025-02-10T21:26:05.402+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.check_file_in_directory scheduled__2025-02-10T21:25:00+00:00 [queued]>
[2025-02-10T21:26:05.404+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-10T21:26:05.418+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_file_in_directory> on 2025-02-10 21:25:00+00:00
[2025-02-10T21:26:05.427+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2839) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T21:26:05.429+0000] {standard_task_runner.py:72} INFO - Started process 2841 to run task
[2025-02-10T21:26:05.429+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag', 'check_file_in_directory', 'scheduled__2025-02-10T21:25:00+00:00', '--job-id', '1103', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmp03kz0fmh']
[2025-02-10T21:26:05.431+0000] {standard_task_runner.py:105} INFO - Job 1103: Subtask check_file_in_directory
[2025-02-10T21:26:05.484+0000] {task_command.py:467} INFO - Running <TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.check_file_in_directory scheduled__2025-02-10T21:25:00+00:00 [running]> on host b6443be51f35
[2025-02-10T21:26:05.593+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag' AIRFLOW_CTX_TASK_ID='check_file_in_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T21:25:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T21:25:00+00:00'
[2025-02-10T21:26:05.595+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-10T21:26:05.596+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-10T21:26:05.597+0000] {logging_mixin.py:190} INFO - Current task name:check_file_in_directory state:running start_date:2025-02-10 21:26:05.394349+00:00
[2025-02-10T21:26:05.597+0000] {logging_mixin.py:190} INFO - Dag name:IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag and current dag run status:running
[2025-02-10T21:26:05.598+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T21:26:05.599+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-10T21:26:05.600+0000] {logging_mixin.py:190} INFO - ca marche /opt/***/files/in ['.csv']
[2025-02-10T21:26:05.605+0000] {python.py:240} INFO - Done. Returned value was: verify_file_reliability
[2025-02-10T21:26:05.606+0000] {branch.py:38} INFO - Branch into verify_file_reliability
[2025-02-10T21:26:05.608+0000] {skipmixin.py:230} INFO - Following branch ('verify_file_reliability',)
[2025-02-10T21:26:05.621+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-10T21:26:05.659+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-10T21:26:05.660+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag, task_id=check_file_in_directory, run_id=scheduled__2025-02-10T21:25:00+00:00, execution_date=20250210T212500, start_date=20250210T212605, end_date=20250210T212605
[2025-02-10T21:26:05.679+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-10T21:26:05.680+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-10T21:26:05.681+0000] {logging_mixin.py:190} INFO - Dag name:IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag queued_at:2025-02-10 21:26:00.031456+00:00
[2025-02-10T21:26:05.682+0000] {logging_mixin.py:190} INFO - Task hostname:b6443be51f35 operator:BranchPythonOperator
[2025-02-10T21:26:05.725+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-10T21:26:05.796+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-10T21:26:05.800+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-10T21:26:52.304+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T21:26:52.446+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.check_file_in_directory scheduled__2025-02-10T21:25:00+00:00 [queued]>
[2025-02-10T21:26:52.458+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.check_file_in_directory scheduled__2025-02-10T21:25:00+00:00 [queued]>
[2025-02-10T21:26:52.459+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-10T21:26:52.474+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_file_in_directory> on 2025-02-10 21:25:00+00:00
[2025-02-10T21:26:52.482+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2877) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T21:26:52.484+0000] {standard_task_runner.py:72} INFO - Started process 2879 to run task
[2025-02-10T21:26:52.485+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag', 'check_file_in_directory', 'scheduled__2025-02-10T21:25:00+00:00', '--job-id', '1109', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmplxz_3r06']
[2025-02-10T21:26:52.487+0000] {standard_task_runner.py:105} INFO - Job 1109: Subtask check_file_in_directory
[2025-02-10T21:26:52.544+0000] {task_command.py:467} INFO - Running <TaskInstance: IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag.check_file_in_directory scheduled__2025-02-10T21:25:00+00:00 [running]> on host b6443be51f35
[2025-02-10T21:26:52.635+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag' AIRFLOW_CTX_TASK_ID='check_file_in_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T21:25:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T21:25:00+00:00'
[2025-02-10T21:26:52.637+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-10T21:26:52.638+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-10T21:26:52.639+0000] {logging_mixin.py:190} INFO - Current task name:check_file_in_directory state:running start_date:2025-02-10 21:26:52.448728+00:00
[2025-02-10T21:26:52.639+0000] {logging_mixin.py:190} INFO - Dag name:IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag and current dag run status:running
[2025-02-10T21:26:52.640+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T21:26:52.641+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-10T21:26:52.642+0000] {logging_mixin.py:190} INFO - ca marche /opt/***/files/in ['.csv']
[2025-02-10T21:26:52.644+0000] {python.py:240} INFO - Done. Returned value was: verify_file_reliability
[2025-02-10T21:26:52.645+0000] {branch.py:38} INFO - Branch into verify_file_reliability
[2025-02-10T21:26:52.646+0000] {skipmixin.py:230} INFO - Following branch ('verify_file_reliability',)
[2025-02-10T21:26:52.662+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-10T21:26:52.694+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-10T21:26:52.695+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag, task_id=check_file_in_directory, run_id=scheduled__2025-02-10T21:25:00+00:00, execution_date=20250210T212500, start_date=20250210T212652, end_date=20250210T212652
[2025-02-10T21:26:52.708+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-10T21:26:52.709+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-10T21:26:52.711+0000] {logging_mixin.py:190} INFO - Dag name:IMPORTATION_ECRITURE_COMPTABLES_ARTICLE_dag queued_at:2025-02-10 21:26:47.474496+00:00
[2025-02-10T21:26:52.712+0000] {logging_mixin.py:190} INFO - Task hostname:b6443be51f35 operator:BranchPythonOperator
[2025-02-10T21:26:52.742+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-10T21:26:52.776+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-10T21:26:52.780+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
